server/chat-plugins/info.js

```diff
@@ -2380,6 +2386,44 @@ const commands = {
 	],
 };
 
+/** @type {PageTable} */
+const pages = {
+	punishments(query, user, connection) {
+		let buf = "";
+		this.extractRoom();
+		if (!user.named) return Rooms.RETRY_AFTER_LOGIN;
+		buf += `|title|Punishments\n|pagehtml|<div class="pad"><h2>List of active punishments:</h2>`;
+		if (!user.can('mute')) {
```

Comment:
Are we still suppressing IPs from drivers?

Reply:
WDYM? Drivers can't use `/ip` and the list doesn't show IP's.